lib/main.rb

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
```

Comment:
## [OPTIONAL] 
* I would suggest you update your readme on how to set up your app properly as this line `#!/usr/bin/env bash`  was giving me errors, only when i removed that line was i able to run your app which was very nice and user friendly 🤗 
![image](https://user-images.githubusercontent.com/52497246/189271487-ad725ed4-9cb9-42df-8371-577ae37484b9.png)

Reply:
Thank you for noticing it. I checked that and it was because I typed the incorrect environment. The line should have been as follows:

`#!/usr/bin/env ruby`

I just changed that in a new commit. Thank you again. Best regards and happy coding!!